src/apps/media/src/app/components/MediaDetailsModal.js

```diff
@@ -105,28 +105,24 @@ export const MediaDetailsModal = memo(function MediaDetailsModal(props) {
             placeholder={"Image Filename"}
             onChange={(val) => setFilename(val)}
           />
-          {/* <FieldTypeText
-            className={styles.Field}
-            name="alt"
-            label={
-              <label>
-                <Infotip title="Edit Alt Attribute " />
-                &nbsp;Alt Attribute
-              </label>
-            }
-            placeholder={"Alt Attribute"}
-          /> */}
 
-          <dl className={styles.DescriptionList}>
```

Comment:
Why are we switching away from `dl` with `dt` and `dd`?

Reply:
Since we read left to right I felt the button nested under ZUID title didn't align nicely.  
 
Screen shot of what it did look like
<img width="232" alt="Screen Shot 2021-09-14 at 2 11 03 PM" src="https://user-images.githubusercontent.com/22800749/133334433-69150007-3fca-47a1-a834-6ca25e86a0b6.png">

Reply:
Why does the styling necessitate a markup change?

Reply:
@shrunyan  I couldn't keep the Description List element tag side by side ex: `zuid: button`, do you want me to keep the  Description List element as is like the image above?

Reply:
I would like us to use semantically meaningful markup when possible. I feel `dl` is an appropriate tag for this scenario. If the visual design of the term and detail being inline is what we want we need to find a way to accomplish this with css or additional markup that lets us achieve the effect without losing semantics.

Reply:
@shrunyan brought back the description list semantics, making the `dl` inline goes against the the purpose of the semantics so leaving as is. I guess it keeps consistency with the remainder of the medial model.

<img width="543" alt="Screen Shot 2021-09-15 at 1 27 55 PM" src="https://user-images.githubusercontent.com/22800749/133504708-1e20fda4-a5f8-47ed-a097-00d704111cf5.png">